src/hypertable.c

```diff
@@ -86,25 +98,25 @@ hypertable_permissions_check(Oid hypertable_oid, Oid userid)
 	if (!has_privs_of_role(userid, ownerid))
 		ereport(ERROR,
 				(errcode(ERRCODE_INSUFFICIENT_PRIVILEGE),
-				 errmsg("permission denied for relation \"%s\"",
+				 errmsg("permission denied for hypertable \"%s\"",
```

Comment:
Chatted in person, but to immortalize here: we decided that since PG was switching to make these calls more specific in 11, we'd do it here as well, and even backport, mostly because otherwise regression tests that otherwise don't need version-specific out files would begin requiring them.